Cannot find module 'pg-native'

Getting this warning message when importing the package with commonjs

`const createPostgresSubscriber = require('pg-listen')`

Getting console message: `Cannot find module 'pg-native'`

Everything seems to be working though. Maybe some problem in dependencies?

Using version pg-listen@1.2.0

+1 here

const subscriber = createSubscriber({
  connectionString: process.env.POSTGRES_URL
}, {
  native: false
})

====> Cannot find module 'pg-native'

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,8 +1,10 @@
 import createDebugLogger from "debug"
 import EventEmitter from "events"
-import * as pg from "pg"
 import format from "pg-format"
 import TypedEventEmitter from "typed-emitter"
+
+// Need to require `pg` like this to avoid ugly error message (see #15)
+import pg = require("pg")
 
 const connectionLogger = createDebugLogger("pg-listen:connection")
 const notificationLogger = createDebugLogger("pg-listen:notification")

